server/dash/dashboards/neighborhood.py

```diff
@@ -39,27 +42,80 @@ def populate_options():
         )
     return values
 
+print(populate_options())
+
+layout = html.Div([
+    html.H1(title),
+    dcc.Dropdown(
+        id='council_list',
+        clearable=False,
+        value="Arleta",
+        placeholder="Select a neighborhood",
+        options=populate_options()
+    ),
+    dcc.Dropdown(
+        id='data_type',
+        clearable=False,
+        placeholder="Select a timeframe",
+        options=[
+            {'label': 'Compare to: Daily Council Average', 'value': 'nc_avg'},
+            {'label': 'Compare to: Weekly Council Average', 'value': 'nc_avg_wk'},
+            {'label': 'Compare to: Monthly Council Average', 'value': 'nc_avg_Mo'},
+        ],
+
+        value='nc_avg'
+    ),
+    dcc.Graph(
+        id='graph1',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    ),
+
+    dcc.Graph(
+        id='graph2',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    )
+])
+
 
 # Define callback to update graph
 @app.callback(
     Output('graph1', 'figure'),
-    [Input("council_list", "value")]
+    [Input("council_list", "value"),
+    Input("data_type", "value")]
 )
-def update_figure(selected_council):
-
+def update_figure(selected_council, select_timeframe):
     neighborhood_sum_df = df[df.council_name == selected_council].groupby(['created_date']).agg('sum').reset_index()  # noqa
+
     total_sum_df = df.groupby(['created_date']).agg('sum').reset_index()
+
     total_sum_df["nc_avg"] = total_sum_df["counts"] / 99
```

Comment:
While you're here, could you create a global variable for this at the top of the file? Following the GLOBAL_CONSTANT_NAME convention. https://google.github.io/styleguide/pyguide.html#316-naming

Reply:
I have change this code. I am not clear what would benefit from being a global variable here. The new pull request has simpler code. I think it could probably be simplified yet using a long table instead of the current format.

Reply:
Apologies, I was talking specifically about `99`. Magic numbers are discouraged. See this stackoverflow [answer](https://stackoverflow.com/a/47902). It's in Java, but the same idea applies to all languages.

Reply:
That's a great convention, thanks for pointing this out. I was actually confused at first as to what 99 was. Partially this is why this ticket was first generated as it was not clear what the graph represented. Hopefully, now it will be more clear.